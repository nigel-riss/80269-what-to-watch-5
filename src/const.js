const FilmTab = {
  OVERVIEW: `Overview`,
  DETAILS: `Details`,
  REVIEWS: `Reviews`,
};

const RatingLimits = {
  BAD: 3,
  NORMAL: 5,
  GOOD: 8,
  VERY_GOOD: 10,
};

const HOVER_PLAY_DELAY = 1000;
const FILM_CARD_HEIGHT = 175;
const FILM_CARD_WIDTH = 280;
const FILMS_SHOWN_PER_CLICK = 8;

export {
  FILM_CARD_HEIGHT,
  FILM_CARD_WIDTH,
  FILMS_SHOWN_PER_CLICK,
  FilmTab,
  HOVER_PLAY_DELAY,
  RatingLimits,
};
