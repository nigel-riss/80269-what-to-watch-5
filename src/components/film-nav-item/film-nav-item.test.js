import React from 'react';
import renderer from 'react-test-renderer';
import FilmNavItem from './film-nav-item.jsx';
import {FilmTabName} from '../../const.js';


const noop = () => {};

it(`Should FilmNavItem render correctly`, () => {
  const tree = renderer
    .create(
        <FilmNavItem
          isActive={true}
          name={FilmTabName.OVERVIEW}
          onClick={noop}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
