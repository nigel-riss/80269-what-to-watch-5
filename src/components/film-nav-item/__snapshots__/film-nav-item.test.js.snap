// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmNavItem render correctly 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
  onClick={[Function]}
>
  <a
    className="movie-nav__link"
    href="#"
  >
    Overview
  </a>
</li>
`;
